Classes/DateTimeParsers.swift

```diff
@@ -55,15 +53,8 @@ final class GPXDateParser {
         components.hour = hour.pointee
         components.month = month.pointee
         components.second = second.pointee
-        
```

Comment:
The tests all crash for me because the `GPXTest-TrackPointOnly` bundle resource isn't found. How do I resolve this so that I can reproduce the actual test failure?

Reply:
Hi @Gerzer, 

The files are in the Test directory https://github.com/vincentneo/CoreGPX/tree/master/Example/Tests

Reply:
The issue is that some of the calendar functions that you’re using aren’t available on Linux. Would it be acceptable to keep your existing code but to fall back on the less sophisticated code when compiling for Linux? The tests would pass on macOS, but they would continue to fail on Linux. I could modify those tests as well as the documentation to reflect the differences in behavior. From what I can tell, the issue occurs when trying to parse non-standard date strings.

Reply:
The current date parser was used as the default was fairly slow, when considering the scale. (like parsing 1000s of dates) 

I reckon we could do a fallback to DateFormatter for linux? (assuming DateFormatter is available?)

Reply:
I apologize for the delay on moving forward with this. I was in a position for a few months in which I wasn’t able to contribute to open-source projects. However, that situation is over, so I’ll follow up here soon.

Reply:
It's alright, welcome back!